Multisig/Data/Services/Notification Service/UnregisterNotificationTokenRequest.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  UnregisterNotificationTokenRequest.swift
+//  Multisig
+//
+//  Created by Moaaz on 8/5/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+struct UnregisterNotificationTokenRequest: JSONRequest {
+    let deviceID: UUID
+    let address: String
+
+    var httpMethod: String { return "DELETE" }
+    var urlPath: String { return "/api/v1/notifications/devices/\(deviceID)/safes/\(address)/" }
+
+    typealias ResponseType = Response
+
+    init(deviceID: UUID, address: Address) {
+        self.address = address.checksummed
+        self.deviceID = deviceID
+    }
+
+    struct Response: Decodable {
```

Comment:
this can be renamed to "EmptyResponse", and in the body added comment "// empty"